www/js/app.js

```diff
@@ -601,6 +624,21 @@ function focusPrefixOnHomeKey (event) {
         }, 0);
     }
 }
+// Verify the source for a loaded archive
+async function verifyLoadedArchive () {
+    if (selectedArchive.isReady() && !(settingsStore.getItem("trustedZimFiles").includes(selectedArchive.file.name)) && params.sourceVerification) {
+        const response = await uiUtil.systemAlert('Is the loaded ZIM archive from a trusted source?\n If not, you can still read the ZIM file in Safe Mode (aka JQuery mode). Closing this window also opens the file in Safe Mode. This option can be disabled in Expert Settings', 'Security alert!', true, 'Open in Safe Mode', 'Trust Source');
```

Comment:
Strings need to be put into a translation-ready format. Look at some other examples of `uiUtil.systemAlert()` to see how to do this. You need to do it for each of the strings here that will need to be translated, i.e. all four strings.